src/test/resources/skatTest.R

```diff
@@ -0,0 +1,47 @@
+# Charlie Colley
```

Comment:
lol no problem, I've got a deal with some folks who are going to make a pyramid for me anyway.